client/src/components/Cart/Cart.css

```diff
@@ -30,46 +54,116 @@
   justify-self: space-around;
 }
 
-.quantity {
-  grid-area: p;
+.sizeBox {
+  grid: p;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(70px, 1fr));
+  border: 1px solid#ebedee;
+  border-left: 1px solid #ebedee;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+}
+
+.sizeBtn {
+  background-color: white;
+  border: 1px solid#ebedee;
+  border-left: none;
+  align-items: center;
   box-sizing: border-box;
-  cursor: cursor;
   height: 40px;
+  font-size: 13px;
+  margin-bottom: -1px;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  letter-spacing: 0.2px;
+  border-color: #ebedee;
+  /* font-weight: 700; */
+}
+
+.sizeBtn:hover {
+  background: black;
+  color: white;
+  border-color: #ebedee;
+}
+
+.sizeBtn:focus {
+  background: black;
+  color: white;
+  border-color: #ebedee;
+}
```

Comment:
Since these have the same properties, you can also combine the class names with a comma

Reply:
nice thank you!